pennylane/collections.py

```diff
@@ -467,9 +520,29 @@ def evaluate(self, args, kwargs):
             list: the results from each QNode
         """
         results = []
+        _async = kwargs.pop("parallel", False)
```

Comment:
I would consider renaming this to `_parallel` so that the name is in direct correspondence with the kwarg name.

Reply:
Done!